types/three/src/renderers/WebGLRenderer.d.ts

```diff
@@ -430,6 +433,14 @@ export class WebGLRenderer implements Renderer {
      */
     copyTextureToTexture(position: Vector2, srcTexture: Texture, dstTexture: Texture, level?: number): void;
 
+    copyTextureToTexture3D(
+        sourceBox: Box3,
+        position: Vector3,
+        srcTexture: Texture,
+        dstTexture: DataTexture3D,
```

Comment:
Can you provide a source for why this should be `DataTexture3D` instead of `Texture`?

Reply:
I'm going off the example here, plus the name of the function made it seem logical, but I can just put it to `Texture`

Reply:
Looks like it supports `DataTexture3D` and `DataTexture2DArray` based on [this](https://github.com/mrdoob/three.js/commit/b5ff213#diff-fc94afb5da943fea9e01a4a16adf82b5433aa6d61af5855f296158a98ed4dcfdR2018), so we can go ahead and narrow it to those types if you want to.

Reply:
I've changed this